lib/ex_unit/lib/ex_unit.ex

```diff
@@ -182,6 +182,8 @@ defmodule ExUnit do
           time = ExUnit.Server.modules_loaded()
           config = persist_defaults(configuration())
           %{failures: failures} = ExUnit.Runner.run(config, time)
```

Comment:
There is also a `run` function in this module and I think we should call the after_run in there too. So maybe we should unify it in a private function?